plugins/reference-assets/android/src/androidTest/java/com/intuit/playerui/android/reference/assets/info/InfoTest.kt

```diff
@@ -28,7 +28,8 @@ class InfoTest : AssetTest("reference-assets") {
         }
 
         action?.let {
-            infoActions[action.ordinal].shouldBeView<Button> {
+            val buttonOrdinal = if (action.ordinal != 1) 0 else action.ordinal
```

Comment:
Not sure I understand this one

Reply:
this is because there's a new enum added and the old logic relying on the ordinal was no longer functional